configure.ac

```diff
@@ -5,6 +5,12 @@ AC_PROG_CC
 
 AC_PATH_PROG(APXS, 'apxs')
 
+if test x"$APXS" = x"" ; then
+  AC_CHECK_FILE([/etc/redhat-release],
```

Comment:
Eh, I figured if we could make it easier, why not? `/etc/redhat-release` exists on el6 systems